addOns/fuzz/src/main/javahelp/org/zaproxy/zap/extension/fuzz/resources/help/contents/api.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+    <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+    <TITLE>
+        Fuzzing API
+    </TITLE>
+</HEAD>
+<H1>Fuzzing API</H1>
+<p>
+    The API has been designed to be used by people with a range of security experience, for
+    example “simpleHttpFuzzer” is a basic fuzzer fuzzes one “Fuzz location”, extracting payloads
+    from one file, and returns the fuzzerId.
+</p>
+<h3>Homepage</h3>
+<p><a href="https://github.com/zaproxy">https://github.com/zaproxy</a><br>
+    Documentation can be found at:<br>
+    <a href="https://github.com/zaproxy/zap-api-docs/fuzz">https://github.com/zaproxy/zap-api-docs/fuzz</a>
+    <br>
+</p>
+<h3>Message Processors</h3>
+<p>
+    The default processor enabled by default for all requests is “Request Content-Length Updater”
+    which updates the content-length field according to the attack payload. Option to add more
+    processors will be added in future releases.
+</p>
+<h3>Fuzzer Options</h3>
+<p>
+    Currently these fuzzer options can be changed through the FuzzAPI but in future releases
+    more options will be added.
+</p>
+<ul>
+    <li>Retries on IO error</li>
+    <li>Max. error allowed</li>
+    <li>Payload Replacement Strategy</li>
+    <li>Concurrent scanning threads per scan</li>
+    <li>Delay when fuzzing (in milliseconds)</li>
+</ul>
+Note: options changed after the fuzzer has been initialised will remain same for that fuzzer.
+<h3>Simple Http Fuzzer</h3>
+<p>The simpleHttpFuzzer fuzzes one fuzz location, with inputs</p>
+<table style="width:100%" border="1px solid black" border-collapse="collapse">
+    <tr>
+        <th>Parameter</th>
+        <th>Type</th>
+        <th>Example</th>
+    </tr>
+    <tr>
+        <td>messageId </td>
+        <td>int</td>
+        <td>7</td>
+    </tr>
+    <tr>
+        <td>requestLocation</td>
+        <td>string
+        </td>
+        <td>“body” or “header”</td>
+    </tr>
+    <tr>
+        <td>fuzzLocation</td>
+        <td>string (“int start: int end”)</td>
+        <td>8:15 or 12:18</td>
+    </tr>
+    <tr>
+        <td>payloadPath</td>
+        <td>string</td>
+        <td>“/home/users/payloads.txt”</td>
+    </tr>
+</table>
+<br>
+The parameter fuzzLocation is the start and end number of the character which is replaced
+by the payload.<br>
+For example:
+<p style="font-family:'Courier 10 Pitch',cursive">POST /cgi-bin/process.cgi HTTP/1.1
```

Comment:
This should probably use `<TT>` or `<code>`... @thc202 what needs to be used here to parse properly for the github wiki pages?

Reply:
Maybe `<pre>` instead of leading the lines with `<br>` (which seems really strange)?

Reply:
Yeah, better to use `<code>` and `<pre>` instead of changing font and adding all the `br` tags.